lib/tasks/custom_seed.rake

```diff
@@ -0,0 +1,12 @@
+# Thanks to https://stackoverflow.com/questions/19872271/adding-a-custom-seed-file/31815032#31815032
+namespace :db do
+  namespace :seed do
+    Dir[Rails.root.join("db/seeds/*.rb")].each do |filename|
```

Comment:
Would you mind using `Dir.glob` instead of `Dir[]`? They are aliases, but I find `glob` is a lot more intention revealing. 

That said, is it really necessary to specify a task for each file? Couldn't a single task with an argument also do the trick?

```ruby
# lib/tasks/e2e_seed.rake
SEED_DIR = Rails.root.join("spec/support/seeds/e2e/")

namespace :db
  namespace :seed do
    desc "Seed data for e2e tests"
    task :e2e, [:file] do |_t, args|
      filename = SEED_DIR.join("#{args[:file]}.rb")
      load(filename) if File.exist?(filename)
    end
  end
end
```

(This code is  untested, I wrote it directly in the Github comment, but you get the idea)

Then instead of calling e.g. `rake db:seed:admin_login_setup` you'd call `rake db:seed:e2e[admin_login_setup]`.